wiki/Recipes.md

```diff
@@ -8,7 +8,7 @@ Recipes are often just procedures, or a set of instructions that will produce a
 
 ## Software Requirements Specification (SRS) Recipe
 
-Currently, the most-used recipe in Drasil is one to generate Software Requirements Specification (SRS), which can be found in the `drasil-docLang` package. It has gone through many iterations of refinement and rationalization, and we have arrived at a point where creating a new physics-based example only requires inputting the necessary knowledge and organizing functions created in the document language. This recipe takes in all of its input from [`SystemInformation`](https://jacquescarette.github.io/Drasil/docs/full/drasil-database-0.1.1.0/Database-Drasil.html#t:SystemInformation) and [`PrintingInformation`](https://jacquescarette.github.io/Drasil/docs/full/drasil-database-0.1.1.0/Database-Drasil.html#t:PrintingInformation) which are given to the [`gen`](https://jacquescarette.github.io/Drasil/docs/full/drasil-database-0.1.1.0/Database-Drasil.html#v:gen) function. And that function formats the information and prepares it for the printers, which then gives us the formatted SRS documents in the manner we desire.
+Currently, the most-used recipe in Drasil is one to generate Software Requirements Specification (SRS), which can be found in the `drasil-docLang` package. It has gone through many iterations of refinement and rationalization, and we have arrived at a point where creating a new physics-based example only requires inputting the necessary knowledge and organizing functions created in the document language. This recipe takes in all of its input from [`System`](https://jacquescarette.github.io/Drasil/docs/full/drasil-database-0.1.1.0/Database-Drasil.html#t:SystemInformation) and [`PrintingInformation`](https://jacquescarette.github.io/Drasil/docs/full/drasil-database-0.1.1.0/Database-Drasil.html#t:PrintingInformation) which are given to the [`gen`](https://jacquescarette.github.io/Drasil/docs/full/drasil-database-0.1.1.0/Database-Drasil.html#v:gen) function. And that function formats the information and prepares it for the printers, which then gives us the formatted SRS documents in the manner we desire.
```

Comment:
Ah, it looks like I forgot to update this URL when I was moving around `SystemInformation`. Do you mind updating the URL that `System` links to, to: https://jacquescarette.github.io/Drasil/docs/full/drasil-sysinfo-0.1.1.0/SysInfo-Drasil-SystemInformation.html#t:SystemInformation ?